### ISSUE TITLE
Remove dependency on composer/composer

### ISSUE THREAD

**Issue by iamsophiesk** [vijaycs85/drupal-quality-checker#18]
This is a Composer package and cannot be installed unless it's through composer. It adds an extra dependency to a Drupal project which doesn't seem to be required. It would be good to remove this dependency if possible.

**Comment by iamsophiesk** [comment]
This seems more pertinent now that Composer 2 is out - we can't upgrade this plugin because of a dependency on Composer 1.

**Comment by ptmkenny** [comment]
Yes, I ran into the same issue today when trying to use composer 2 to update a site that had this installed.  Please consider releasing a 1.2.x (D8) and 1.3.x (D9) version with support for composer 2.

**Comment by gugerard** [comment]
Hi there, Hi @vijaycs85  :-) ,

Same issue here with Drupal 9 and composer 2, after deleting the line :
    "composer/composer": "~1.0",

In composer.json it works as expected.

Pull request linked to this issue : https://github.com/vijaycs85/drupal-quality-checker/pull/29

**Comment by vijaycs85** [comment]
thanks you. merged and released 1.4.0.

**Comment by gugerard** [comment]
Awesome.
